### ISSUE TITLE
Mac OS: physical memory calc

### ISSUE THREAD

**Issue by gedw99** [jech/storrent#9]
Mac OS

```sh
2023/12/21 19:21:14 Couldn't determine physical memory: cannot compute physical memory on this platform
```

Need to see how we can do this.

**Comment by jech** [comment]
Could you please run the following and report the output?
```
package main

import (
	"log"

	"golang.org/x/sys/unix"
)

func main() {
	v, err := unix.SysctlUint64("hw.memsize")
	if err != nil {
		log.Fatalf("Sysctl: %v", err)
	}

	log.Println("Memory size is %v", v)
}
```

**Comment by gedw99** [comment]
Memory size is %v 17179869184

**Comment by jech** [comment]
That's on a system with 16GB physical memory, right?

**Comment by jech** [comment]
Fixed in e14006b.

**Comment by gedw99** [comment]
thanks @jech
